deploy/ansible/provision.yml

```diff
@@ -14,159 +16,30 @@
         image: "{{ ec2_image }}"
         vpc_subnet_id: "{{ ec2_subnet_id }}"
         region: "{{ ec2_region }}"
-        instance_tags: '{"Type":"{{ec2_instance_type}}", "Role":"{{ec2_tag_Role}}"}'
+        instance_tags: '{"Type":"{{ec2_instance_type}}", "Role":"{{nucypher_network_name}}"}'
         assign_public_ip: yes
         wait: true
         exact_count: "{{ ec2_count }}"
         count_tag:
-          Role: "{{ ec2_tag_Role }}"
+          Role: "{{ nucypher_network_name }}"
         volumes:
           - device_name: /dev/xvda
             volume_type: gp2
             volume_size: "{{ ec2_volume_size }}"
             delete_on_termination: true
-      register: ec2
+      register: nucypher_swarm
```

Comment:
Why this name?

Reply:
Yeah, I thought "fleets" was our thing.

Reply:
oh yeah, a bit of unintentional name-shadowing here

Reply:
:heavy_check_mark: